controller/instance_manager_controller.go

```diff
@@ -473,21 +473,38 @@ func (imc *InstanceManagerController) syncInstanceManagerPDB(im *longhorn.Instan
 		if imPDB == nil {
 			return nil
 		}
-		canDelete, err := imc.canDeleteInstanceManagerPDB(im)
+		imc.logger.Debugf("Node %v is marked unschedulable, try to remove %v PBD", imc.controllerID, im.Name)
```

Comment:
We should separate cases for enabling CA or disabling CA instead of mixing the implementation together. 

```go
if unscheduled {
    if enabled {
        // the new case for CA awareness 
    } else {
        // the same as before
    }
}
```

Reply:
Fixed.